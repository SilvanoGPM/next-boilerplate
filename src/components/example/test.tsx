import { render, screen } from '@testing-library/react';

import { Example } from '.';

describe('<Example />', () => {
  test('deve renderizar o componente de exemplo', () => {
    const { container } = render(<Example />);

    const expected = screen.getByRole('heading', {
      name: /componente de exemplo/i,
    });

    expect(expected).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
