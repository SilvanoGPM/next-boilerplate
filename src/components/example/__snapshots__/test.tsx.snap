// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Example /> deve renderizar o componente de exemplo 1`] = `
<h1>
  Componente de Exemplo
</h1>
`;
